packages/data-bite/src/components/EditorPanel.js

```diff
@@ -32,7 +32,7 @@ const TextField = memo(({label, section = null, subsection = null, fieldName, up
     if('number' !== type || min === null){
       setValue(e.target.value);
     } else {
-      if(!e.target.value || ( parseFloat(min) <= parseFloat(e.target.value ) & parseFloat(max) >= parseFloat(e.target.value))) {
+      if(!e.target.value || ( parseFloat(min) <= parseFloat(e.target.value ) && parseFloat(max) >= parseFloat(e.target.value))) {
```

Comment:
@Atash3000 - Can you explain the change from bitwise & to logical && here?

Reply:
I was having issue on when we text on Message field.when we clear text field then we dont see anything.i assumed it could be something there.but i dont why to be honest.